neo4j-examples/paradise-papers-django
# Separate the entities, create a class to detail the nodes, update the classes of the view, update the templates and cleanup

Closes [Ticket](https://trello.com/c/xgMjZqIM/33-as-a-developer-i-want-to-separate-model-files-so-that-it-doesnt-get-too-big)


The model file was getting too big and every new entity will make it bigger so we decided to separate on a file for entity and a helper file for all functions related to the model

Still need some refactor but is a start 😅 

## Testing

Steps to manually verify the change:
1. _Setup_: Set the virtual env, install the requirements and run your local server
2. _Exercise_: How to run the environment and example link: [Test Link](http://127.0.0.1:8000/search)
3. _Verify_: Should function normally
